Makefile

```diff
@@ -76,6 +76,11 @@ shell:
 	-v ${PWD}:/go/src/github.com/kris-nova/kubicorn \
 	--rm ${SHELL_IMAGE} /bin/bash
 
+lint:
+	which golint > /dev/null || go get -u github.com/golang/lint/golint...
```

Comment:
There are three dots on the end which shouldn't be there. You can just use `go get -u github.com/golang/lint/golint`